tests/test_download.py

```diff
@@ -53,6 +52,30 @@ def evr(self):
             return '%s-%s' % (self.version, self.release)
 
     @property
+    def source_debug_name(self):
+        """
+        returns name of debuginfo package for source package of given package
+        e.g. krb5-libs -> krb5-debuginfo
+        """
+        return "{}-debuginfo".format(self.source_name)
+
+    @property
+    def source_name(self):
+        """"
+        returns name of source package
+        e.g. krb5-libs -> krb5
+        """
+        if self.sourcerpm is not None:
+            # trim suffix first
+            srcname = dnf.util.rtrim(self.sourcerpm, ".src.rpm")
```

Comment:
http://developers.redhat.com/blog/2014/12/10/how-to-package-proprietary-software/

If you put `NoSource: X` into spec file - rpmbuild will create `.nosrc.rpm`. It's not specific to your PR, just made note about thing.